This commit adds a new package file for the SDL2_ttf library, which p…

georgemalerbo opened the pull request:
…rovides support for rendering TrueType fonts in SDL2 applications

Commit:
This commit adds a new package file for the SDL2_ttf library, which provides support for rendering TrueType fonts in SDL2 applications

amklinv commented:
Please run `spack style -f` to fix the format.

Commit:
Formatted recipe using spack style

bernhardkaindl reviewed (commented):
@georgemalerbo Do you really use these older 2.0 versions?

As this is a new package, I'd say:

If you don't need all of these older versions, I'd only add the newest or newest two versions, the fewer the better.

If you'd be available by mail for notifications on PRs on this new package:

Would you, when you are at it, consider to add yourself as maintainer like spackbot suggests?

The only consequence is that you get a notification mail when someone submits a PR, and you get to say if you like the change or not.

Commit:
Removed older versions and added maintainers for SDL2_ttf package

georgemalerbo commented:
> @georgemalerbo Do you really use these older 2.0 versions?
> 
> 
> 
> As this is a new package, I'd say:
> 
> 
> 
> If you don't need all of these older versions, I'd only add the newest or newest two versions, the fewer the better.
> 
> 
> 
> If you'd be available by mail for notifications on PRs on this new package:
> 
> 
> 
> Would you, when you are at it, consider to add yourself as maintainer like spackbot suggests?
> 
> 
> 
> The only consequence is that you get a notification mail when someone submits a PR, and you get to say if you like the change or not.

Thanks for your feedback. I removed the older versions, and added myself as maintainer as well as a co-maintainer who has agreed to take on the role with me.
